setup.py

```diff
@@ -0,0 +1,52 @@
+#!/usr/bin/env python
+
+import os
+import codecs
+from setuptools import (
+    setup,
+    find_packages,
+)
+from tor_core import __version__
+
+
+def long_description():
+    if not (os.path.isfile('README.md') and os.access('README.md', os.R_OK)):
+        return ''
+
+    with codecs.open('README.md', encoding='utf8') as f:
+        return f.read()
+
+
+setup(
+    name='tor_core',
+    version=__version__,
+    description='',
+    long_description=long_description(),
+    url='https://github.com/transcribersofreddit/tor_core',
+    author='Joe Kaufeld',
+    author_email='joe.kaufeld@gmail.com',
+    license='MIT',
+    classifiers=[
+        'Development Status :: 1 - Planning',
```

Comment:
Funny enough, I wrote up the classifiers when I was trying to split up `tor` into separate packages for my initial refactor. This was a leftover from that.

I think I left it in because, despite the version number indicating stability, it seemed like the project was still in its infancy and didn't have a clear-cut API quite yet.